responses/__init__.py

```diff
@@ -809,11 +810,16 @@ def start(self):
         def unbound_on_send(adapter, request, *a, **kwargs):
             return self._on_request(adapter, request, *a, **kwargs)
 
-        self._patcher = std_mock.patch(target=self.target, new=unbound_on_send)
-        self._patcher.start()
+        if not self._patcher:
```

Comment:
Should the function on line 810 only be defined when we're creating a patcher as well?